components/form/introduction.rst

```diff
@@ -163,14 +163,14 @@ component offers a rich integration.
 
 To use the integration, you'll need the ``TwigBridge``, which provides integration
 between Twig and several Symfony components. If you're using Composer, you
-could install the latest 2.3 version by adding the following ``require``
+could install the latest 2.7 version by adding the following ``require``
```

Comment:
The installation instructions in this document should probably be rewritten, but that should be done in a different PR.